website/docs/r/google_folder.html.markdown

```diff
@@ -1,7 +1,7 @@
 ---
 layout: "google"
 page_title: "Google: google_folder"
-sidebar_current: "docs-google-folder"
+sidebar_current: "docs-google-folder-x"
```

Comment:
It does a prefix match when doing highlighting of the current page you are visiting. If you don't put the `-x`, then, when visiting docs-google-folder-iam-policy it will highlight both links in the sidebar which is not desirable.

For instance, the aws provider has this issue for some of their resource:
https://www.terraform.io/docs/providers/aws/d/ami_ids.html

See how `aws_ami` and `aws_ami_ids` links are highlighted in the sidebar. Adding the -x prevents this issue.

Reply:
I'll remember this, thanks!